make unit_test in MacOS encounter "./scripts/generate-protobuf.sh: line 62: shopt: globstar: invalid shell option name" error

### What did you do?

run make unit_test

### What did you expect to see?

all tests pass

### What did you see instead?

make unit_test
scripts/generate-mock.sh
go generate
generate-protobuf
./scripts/generate-protobuf.sh
generate ./proto/EntryProtocol.proto...
generate ./proto/CanalProtocol.proto...
generate ./proto/CraftBenchmark.proto...
generate ./proto/CDCPeerToPeer.proto...
generate ./dm/proto/dmworker.proto...
generate ./dm/proto/dmmaster.proto...
./scripts/generate-protobuf.sh: line 62: shopt: globstar: invalid shell option name
make: *** [generate-protobuf] Error 1

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
master
```

--- a/scripts/generate-protobuf.sh
+++ b/scripts/generate-protobuf.sh
@@ -61,8 +61,8 @@ generate ./dm/pb ./dm/proto/dmworker.proto plugins=grpc,protoc-gen-grpc-gateway=
 generate ./dm/pb ./dm/proto/dmmaster.proto plugins=grpc,protoc-gen-grpc-gateway="$GRPC_GATEWAY"
 
 for pb in $(find cdc -name '*.proto'); do
-    # Output generated go files next to protobuf files.
-    generate ./cdc $pb paths="source_relative"
+	# Output generated go files next to protobuf files.
+	generate ./cdc $pb paths="source_relative"
 done
 
 echo "generate enginepb..."

--- a/scripts/generate-protobuf.sh
+++ b/scripts/generate-protobuf.sh
@@ -59,10 +59,14 @@ generate ./proto/benchmark ./proto/CraftBenchmark.proto
 generate ./proto/p2p ./proto/CDCPeerToPeer.proto plugins=grpc
 generate ./dm/pb ./dm/proto/dmworker.proto plugins=grpc,protoc-gen-grpc-gateway="$GRPC_GATEWAY"
 generate ./dm/pb ./dm/proto/dmmaster.proto plugins=grpc,protoc-gen-grpc-gateway="$GRPC_GATEWAY"
-shopt -s globstar
-for pb in cdc/**/*.proto; do
-	# Output generated go files next to protobuf files.
-	generate ./cdc $pb paths="source_relative"
+# shopt -s globstar
+# for pb in cdc/**/*.proto; do
+# 	# Output generated go files next to protobuf files.
+# 	generate ./cdc $pb paths="source_relative"
+# done
+for pb in $(find cdc -name '*.proto'); do
+    # Output generated go files next to protobuf files.
+    generate ./cdc $pb paths="source_relative"
 done
 
 echo "generate enginepb..."

--- a/scripts/generate-protobuf.sh
+++ b/scripts/generate-protobuf.sh
@@ -59,11 +59,7 @@ generate ./proto/benchmark ./proto/CraftBenchmark.proto
 generate ./proto/p2p ./proto/CDCPeerToPeer.proto plugins=grpc
 generate ./dm/pb ./dm/proto/dmworker.proto plugins=grpc,protoc-gen-grpc-gateway="$GRPC_GATEWAY"
 generate ./dm/pb ./dm/proto/dmmaster.proto plugins=grpc,protoc-gen-grpc-gateway="$GRPC_GATEWAY"
-# shopt -s globstar
-# for pb in cdc/**/*.proto; do
-# 	# Output generated go files next to protobuf files.
-# 	generate ./cdc $pb paths="source_relative"
-# done
+
 for pb in $(find cdc -name '*.proto'); do
     # Output generated go files next to protobuf files.
     generate ./cdc $pb paths="source_relative"

